### ISSUE TITLE
Feature Request: Make Glossaries

### ISSUE THREAD

**Issue by 240ch** [gallandarakhneorg/autolatex#100]
I'd love to have autolatex generate glossaries too either default or with an option "-gloss" or something like that.
The needed command is `makeglossaries [mydoc]`

Thank you very much!

**Comment by Patterner** [comment]
+1

I have at least one LaTeX project which uses a glossary and I would test the solution.

**Comment by gallandarakhneorg** [comment]
Ok. I agree. I push this issue on the task list.

**Comment by gallandarakhneorg** [comment]
The support of makeglossaries is added. It was successfully tested on basic documents. It needs to be tested on real-size documents.

**Comment by 240ch** [comment]
Am I trying to use the option wrong?
```
user@ubuntu:~$ autolatex --version
autolatex 39.0 - Unix platform
(c) 1998-2016 Stephane GALLAND <galland@arakhne.org> (under GPL)
user@ubuntu:~$ autolatex -f doc.tex --gloss
Unknown option: gloss
```

**Comment by gallandarakhneorg** [comment]
The support of makeglossaries is enabled by default. You don't need to pass --gloss.
AutoLaTeX parses your document, detects uses of macros related to makeglossaries, and launch makeglossaries accordingly.
However, event if the module were tootally implemented, I missed to define the command-line options in the main function. It will be fixed soon.

**Comment by gallandarakhneorg** [comment]
The fix for the '--gloss' command line was applied. It will be included into the next planned version: 39.1.
